agent/api/task/task_windows.go

```diff
@@ -91,6 +96,16 @@ func isBareDrive(path string) bool {
 	return false
 }
 
+func isNamedPipesPath(path string) bool {
+	matched, err := regexp.MatchString(`\\{2}\.[\\]pipe[\\].+`, path)
```

Comment:
Just curious to know why using [\\] if it only needs to match '\'.

Reply:
I was escaping it because it is literally a "." in the regular expression and not just any character.

Reply:
Sorry. The markdown removes some characters. I am asking the using of [\\\\], since we only want to match '\\', which is the only character. Like `\\{2}\.\\pipe\\.+` actually does the same thing.